scripts/cnv_wdl/somatic/cnv_somatic_pair_workflow.wdl

```diff
@@ -673,6 +675,7 @@ task CallCopyRatioSegments {
 
     output {
         File called_copy_ratio_segments = "${entity_id}.called.seg"
+        File called_copy_ratio_legacy_segments = "${entity_id}.called.seg.igv.seg"
```

Comment:
This filename is pretty gross...can we just make the extension `called.igv.seg`?

Reply:
Done